README.md

```diff
@@ -329,6 +330,16 @@ importing. By default, only modules listed under `dependencies` and
 importDevDependencies: true
 ```
 
+### `dangleComma`
```

Comment:
I think it would be more natural for the config option to be named `danglingCommas`. It would make it more consistent with the rest of the config options.